pkgs/by-name/sa/satochip-utils/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  python3Packages,
+  python3,
+  fetchFromGitHub,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "Satochip-Utils";
```

I think it is preferrable to keep the pname in lowercase
```suggestion
  pname = "satochip-utils";
```